com.unity.render-pipelines.high-definition/Runtime/Material/Decal/Decal.shader

```diff
@@ -78,6 +78,8 @@ Shader "HDRP/Decal"
     #pragma shader_feature_local _MATERIAL_AFFECTS_MASKMAP
 
     #pragma multi_compile_instancing
+    // enable dithering LOD crossfade
+    #pragma multi_compile _ LOD_FADE_CROSSFADE
```

Comment:
If I understand the issue correctly this should only be apply with decal mesh right?
or it works with decal projector? if it apply only to decal mesh, then the 
#pragma multi_compile _ LOD_FADE_CROSSFADE
must be move in DecalMeshForwardEmissive and DBufferMesh pass (same for shader graph)

Reply:
Done